scripts/yrd/install

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -e
+
+# get previous location - should be 
+START_DIR="$(pwd)"
+
+BASE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+mkdir "$BASEDIR/tmp"
+cd "$BASEDIR/tmp"
+git clone https://github.com/kpcyrd/yrd.git
+cd yrd
+# install dependencies
+sudo apt-get install python3 python3-pip python3-argh python3-requests
+# install yrd
+sudo -H python3 ./setup.py -v install
+# install and enable the service
+cd "$START_DIR"
+sudo cp yrd.service /etc/systemd/system/
+sudo systemctl enable yrd
```

Comment:
Let's add this before enabling to be safe:

```
sudo systemctl daemon-reload
```

Maybe it's no longer necessary but I have had issues with earlier images.